packages/babel-helper-transform-fixture-test-runner/src/helpers.js

```diff
@@ -11,5 +11,3 @@ export function assertLacksOwnProperty() {}
 export function multiline(arr) {
   return arr.join("\n");
 }
-
-export const assertArrayEquals = assert.deepEqual;
```

Comment:
technically a breaking change but we didn't intend for anyone to use this package

Reply:
Since this PR will land in Babel 7, I think we could even remove also the empty functions from this file.

@devenbansod You missed another usage of `assert` in this file :wink:

Reply:
The methods `assertHasOwnProperty` and `assertLacksOwnProperty` seem to be used in `babel-preset-es2015/test/fixtures/traceur/Classes/InheritanceFromMemberExpression.js` and `babel-preset-es2015/test/fixtures/traceur/Classes/MethodInheritance.js`, though actually since they are empty so are really not testing anything.

Can we replace with following:
```
export function assertHasOwnProperty() {}
```
by 
```
export function assertHasOwnProperty(obj, propertyName) {
     expect(obj).toHaveProperty(propertyName);
}
```

I can't think of functionally equivalent of `assertHasNoOwnProperty` though. Any ideas?
OR should we just remove the usages of these functions?

@nicolo-ribaudo thanks for pointing that out. Just replaced that usage with expect.

Reply:
> ```
> export function assertHasOwnProperty() {}
> ```
>
> by
>
> ```
> export function assertHasOwnProperty(obj, propertyName) {
>      expect(obj).toHaveProperty(propertyName);
> }
> ```

You can directly use expect().toHaveProperty() in those tests, without the `assertHasOwnProperty` function.

> I can't think of functionally equivalent of assertHasNoOwnProperty though. Any ideas?
OR should we just remove the usages of these functions?

You can use something like this:
```js
expect(Object.getOwnPropertyNames(obj)).toHaveLength(0);
```